blocks/editable/index.js

```diff
@@ -58,6 +58,28 @@ function isLinkBoundary( fragment ) {
 		fragment.childNodes[ 0 ].text[ 0 ] === '\uFEFF';
 }
 
+/**
+ * Transforms internal block's representation into an Element.
```

Comment:
Should these transforms be defined elsewhere? As a generic block API, if blocks are aware of this structure via `children` source?